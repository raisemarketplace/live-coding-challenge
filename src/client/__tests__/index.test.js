import React from "react";
import ReactTestRenderer from "react-test-renderer";

import App from "../app";

describe("App", () => {
  test("should be a valid React Element", () => {
    expect(React.isValidElement(<App />)).toBeTruthy();
  });

  test("should render without throwing an error", () => {
    expect(() => ReactTestRenderer.create(<App />)).not.toThrow();
  });
});

describe("App snapshots", () => {
  test("matches the default snapshot", () => {
    const tree = ReactTestRenderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
