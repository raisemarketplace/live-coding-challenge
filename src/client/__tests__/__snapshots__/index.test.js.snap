// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshots matches the default snapshot 1`] = `
<div
  className="app"
>
  <h1>
    Live Coding Challenge
  </h1>
  <div
    className="content"
  >
    <h2>
      Listings
    </h2>
    <h2>
      Cart
    </h2>
  </div>
</div>
`;
